docs/demo/materials/atoms/table.html

```diff
@@ -1,51 +1,199 @@
+---
+name: Table
+title: Table
+notes: |
+    Use `dc-table` to style tables. Use the varians `dc-table--tight`, `dc-table--comfortable` and `dc-table--spacious` for additional padding options.
+---
 <table class="dc-table">
 	<thead class="dc-table__thead">
+		<tr class="dc-table__tr">
+			<th class="dc-table__th" data-col="First Name">First Name</th>
+			<th class="dc-table__th" data-col="Name">Last Name</th>
+			<th class="dc-table__th" data-col="Name">Email</th>
+			<th class="dc-table__th" data-col="Name">Status</th>
+		</tr>
+	</thead>
 
+	<tbody class="dc-table__tbody">
 		<tr class="dc-table__tr">
-			<th class="dc-table__th">Name</th>
-			<th class="dc-table__th">Market</th>
-			<th class="dc-table__th">Gender</th>
-			<th class="dc-table__th">Page URL</th>
-			<th class="dc-table__th">Status</th>
-			<th class="dc-table__th">Last modified</th>
+			<td class="dc-table__td" data-col="First Name">Eileen</td>
+			<td class="dc-table__td" data-col="Last Name">Holland</td>
+			<td class="dc-table__td" data-col="Email">eileen.holland42@example.com</td>
+			<td class="dc-table__td" data-col="Status"><span class="status nowrap status--active">Active</span></td>
+		</tr>
+
+		<tr class="dc-table__tr">
+			<td class="dc-table__td" data-col="First Name">Duane</td>
+			<td class="dc-table__td" data-col="Last Name">Holmes</td>
+			<td class="dc-table__td" data-col="Email">duane.holmes21@example.com</td>
+			<td class="dc-table__td" data-col="Status"><span class="status nowrap  status--active">Active</span></td>
+		</tr>
+
+		<tr class="dc-table__tr">
+			<td class="dc-table__td" data-col="First Name">Kitty</td>
+			<td class="dc-table__td" data-col="Last Name">Murray</td>
+			<td class="dc-table__td" data-col="Email">kitty.murray88@example.com</td>
+			<td class="dc-table__td" data-col="Status"><span class="status nowrap  status--new" status="New">New</span></td>
```

Comment:
@gabriel should we remove or move also status nowrap status--active?
Why not merging this two #90 #89 in one issue and one pull request?